app/javascript/apps/ArtsyPullRequests/ArtsyPullRequests.spec.tsx

```diff
@@ -1,9 +1,7 @@
 import React from "react"
 import { act } from "react-dom/test-utils"
-import { mount } from "enzyme"
+import { render, screen, waitFor } from "@testing-library/react"
 import { ArtsyPullRequests, ArtsyPullRequestsProps } from "./"
-import { PullList } from "./components/PullList"
-import { PullItem } from "./components/PullItem"
```

Comment:
b/c we're not testing implementation details, we don't even need to know these components exist.